logical/auth.go

```diff
@@ -58,7 +58,13 @@ type Auth struct {
 
 	// Alias is the information about the authenticated client returned by
 	// the auth backend
-	Alias *Alias `json:"alias" structs:"alias" mapstructure:"alias"`
+	Alias *Alias `json:"alias" structs:"alias" mapstructure:"alias" structs:"alias"`
```

Comment:
There is a duplicate `structs` tag here.

Reply:
Fixed.